metastore/models.py

```diff
@@ -34,7 +36,7 @@ def build_dsl(kind_params, userid, kw):
     # All Datasets:
     all_datasets = {
         'bool': {
-            'should': [{'match': {kind_params['findability']: 'published'}},
+            'should': [{'match': {kind_params['findability']: False}},
```

Comment:
Yes, I know and there is an issue for that and will fix in #7, for now, I left as it is private/public.